src/cli.ts

```diff
@@ -28,15 +28,18 @@ const options = new Command("terminal-screenshot")
   try {
     const buffer = await renderScreenshot({
       data: options.data || (await getStdinData()),
-      margin: options.margin ? parseInt(options.margin) : (undefined as any),
+      margin: options.margin ? parseInt(options.margin) : undefined,
       fontFamily: options.fontFamily,
       backgroundColor: options.backgroundColor,
       type: options.type,
     });
 
     await writeFile(options.output, buffer);
   } catch (error) {
-    console.error("Error: " + error.message);
+    if (error instanceof Error) {
+      // eslint-disable-next-line no-console
+      console.error("Error: " + error.message);
+    }
```

Comment:
Let's also log the error in case it wasn't a native `Error` instance:

```suggestion
    } else {
      console.error(error);
    }
```